### DISCUSSION TITLE
Glob support - Why did we get rid of it since Keto 0.6 Zanzibar implementation

### DISCUSSION THREAD

**Discussion by amrosebirani** [ory/keto#873] [Q&A] [answered]
We have been using Keto 0.5.6 in our organization since last 2 years.

Recently we are thinking to move our implementation to the latest version of keto.

However I see that the glob support on the subject, actions and resources has been removed now.

So before moving forward with the implementation wanted to understand few things.

1. Is glob not supported in the Zanzibar protocol?
2. Is glob in the roadmap for future versions of keto?
3. If it is not supported and not in the roadmap, how do you suggest we migrate from older glob based policies in 0.5.* to new relation tuple based policies in 0.8.*

**Comment by zepatrik** [comment, marked as answer]
> Is glob not supported in the Zanzibar protocol?

No.

> Is glob in the roadmap for future versions of keto?

No.

> How do you suggest we migrate from older glob based policies in 0.5.* to new relation tuple based policies in 0.8.*

This is hard to tell, as I don't know how your permission model looks like.

Generally speaking, Zanzibar Keto uses a whole different architecture and way it works. There is no 1:1 feature mapping. Glob support was dropped because it is not compatible with that architecture and goals of the project.
You can stay on v0.5.6. It will not receive any features but critical security updates.
To switch, you will have to re-think your permissions and integrations a bit. Happy to help if you give a few insights on your use-case.
